inst/include/dust/gpu/cuda.cuh

```diff
@@ -0,0 +1,28 @@
+#ifndef DUST_CUDA_CUH
+#define DUST_CUDA_CUH
+#include <stdio.h>
+
```

Comment:
Add a note on how this file is used: include at the top of each `.cu` file (not in the `.hpp`/`.cuh` files in which `CUDA_CALL` is actually used, as this gives a compiler error)

Reply:
Sorry, I don't think I understand this. Is how it's being used ok? If so do I just add:

```
// Include this file at the top of each .cu file (not in the .hpp/.cuh files in which CUDA_CALL is actually used, as this gives a compiler error)
```

in this file?

Reply:
Hmm, I have included this in a .hpp file (in order to get the .cu file and the .cpp file as similar as possible). Is this erroring for you?

Reply:
Previously when I did this I got an error because I was ending up with two compile units (which is why I ended up adding relocatable device code, and all the mess that entailed). Perhaps I had included it in both the header and the .cu, not sure. If it's working then no worries